tests/unit/behaviors/TranslatablePageTest.php

```diff
@@ -48,7 +48,7 @@ public function testUseFallback()
         ]);
         $page->translateContext('fr');
         $this->assertEquals('english title', $page->title);
-        $page->noFallbackLocale()->translateContext('fr');
```

Comment:
Please, keep the tests for the deprecated method (until it actually gets removed).